Tests/NotAutoLayoutTests/LayoutTest/ViewLayoutGuidesTest.swift

```diff
@@ -128,7 +128,6 @@ class ViewLayoutGuidesTest: XCTestCase {
 		
 	}
 	
-	@available(iOS 11.0, *)
 	func testSafeAreaGuide() {
 		
 		// TODO: Find a way to add the view controller to the view hierarchy, otherwise the views cannot get the correct safe area settings.
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/el-hoshino/NotAutoLayout/blob/40071718b06cab9d215b6b88d43e2a3ff98465a7/Tests/NotAutoLayoutTests/LayoutTest/ViewLayoutGuidesTest.swift#L133"></span>TODOs should be resolved (Find a way to add the view con…).<br />
<code>todo</code> <code>ViewLayoutGuidesTest.swift:133</code></td>
    </tr>
  </tbody>
</table>